src/thorin/util/utility.h

```diff
@@ -98,7 +98,7 @@ class TaggedPtr {
 private:
 #if defined(__x86_64__) || (_M_X64)
     int64_t ptr_   : 48; // sign extend to make pointer canonical
-    I       index_ : 16;
+    int64_t index_ : 16;
```

Comment:
Using different types in a bitfield makes MSVC do weird padding stuff and the static_assert() fails. In the case of TaggedPtr with int64_t / int in the bitfield sizeof(TaggedPtr) == 16 rather than the desired 8 bytes. Furthermore, TaggedPtr seems to be unused.